src/Sulu/Bundle/ContactBundle/Entity/AccountContact.php

```diff
@@ -69,11 +69,11 @@ public function getId()
     /**
      * Set contact.
      *
-     * @param \Sulu\Bundle\ContactBundle\Entity\Contact $contact
+     * @param \Sulu\Component\Contact\Model\ContactInterface $contact
      *
      * @return AccountContact
      */
-    public function setContact(\Sulu\Bundle\ContactBundle\Entity\Contact $contact)
+    public function setContact(\Sulu\Component\Contact\Model\ContactInterface $contact)
```

Comment:
You could also use the Interface in order to not use the FQN here. Just mentioning because you have done it in other classes already.